src/mesh/generated/meshtastic/telemetry.pb.h

```diff
@@ -77,7 +77,9 @@ typedef enum _meshtastic_TelemetrySensorType {
     /* MLX90614 non-contact IR temperature sensor */
     meshtastic_TelemetrySensorType_MLX90614 = 31,
     /* SCD40/SCD41 CO2, humidity, temperature sensor */
-    meshtastic_TelemetrySensorType_SCD4X = 32
+    meshtastic_TelemetrySensorType_SCD4X = 32,
+    /* ClimateGuard RadSens, radiation, Geiger-Muller Tube */
+    meshtastic_TelemetrySensorType_RADSENS = 33
```

Comment:
Please don't commit any files named 'generated' - these will usually cause merge conflicts and will be overwritten anyway. The build errors can be ignored and will be fixed by the protobuf PR: